src/Data/Distributive.hs

```diff
@@ -72,6 +70,7 @@ import GHC.Generics (U1(..), (:*:)(..), (:.:)(..), Par1(..), Rec1(..), M1(..))
 -- and no extra information to try to merge together.
 --
 class Functor g => Distributive g where
+  {-# MINIMAL distribute | collect #-}
```

Comment:
This pragma was only introduced in GHC 7.8. Make sure to surround it with `#if __GLASGOW_HASKELL__ >= 708 ... #endif`!